packages/auth/src/accessToken/service.ts

```diff
@@ -159,10 +164,11 @@ async function createAccessToken(
 
 async function rotate(
   deps: ServiceDependencies,
-  managementId: string
+  managementId: string,
+  tokenValue: string
 ): Promise<Rotation> {
```

Comment:
What about
```suggestion
): Promise<Token | undefined> {
```
instead of the `success`, and let the calling route format the response body